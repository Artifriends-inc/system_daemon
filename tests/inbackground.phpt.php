<?php

require_once 'tests-config.php';
require_once 'System/Daemon.php';

System_Daemon::setOption("appName", "test");
System_Daemon::setOption("logVerbosity", System_Daemon::LOG_EMERG);

$res = System_Daemon::daemonIsInBackground();
var_dump($res);

?>