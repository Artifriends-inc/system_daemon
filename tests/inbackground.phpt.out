Array
(
    [appName] => test
)
Array
(
    [appName] => test
    [logVerbosity] => 0
)
bool(false)
